src/test/kotlin/com/example/realworldkotlinspringbootjdbc/presentation/FavoriteControllerTest.kt

```diff
@@ -125,4 +127,108 @@ class FavoriteControllerTest {
             }
         }
     }
+
+    class Unfavorite {
+        private val requestHeader = "dummy-authorize"
+        private val pathParamSlug = "dummy-slug"
+        val dummyRegisteredUser = RegisteredUser.newWithoutValidation(
+            UserId(1),
+            Email.newWithoutValidation("dummy@example.com"),
+            Username.newWithoutValidation("dummy-name"),
+            Bio.newWithoutValidation("dummy-bio"),
+            Image.newWithoutValidation("dummy-image"),
+        )
+
+        data class TestCase(
+            val title: String,
+            val useCaseExecuteResult: Either<UnfavoriteUseCase.Error, CreatedArticle>,
+            val expected: ResponseEntity<String>,
+        )
+
+        @TestFactory
+        fun test(): Stream<DynamicNode> {
+            return Stream.of(
+                TestCase(
+                    title = "成功: UnfavoriteUseCase が Unit を返す場合、200 レスポンスを返す",
```

Comment:
[nits]
aaa、すいませｎ

成功 → 正常系
```suggestion
                    title = "正常系: UnfavoriteUseCase が Unit を返す場合、200 レスポンスを返す",
```

Reply:
こちら修正しました。

https://github.com/sunakan/realworld-kotlin-springboot-jdbc/pull/111/commits/054778d72ae1fece301113e7600fb6d5ede17df4